jerry-ext/handler/handler-register.c

```diff
@@ -40,3 +40,113 @@ jerryx_handler_register_global (const jerry_char_t *name_p, /**< name of the fun
 
   return result_val;
 } /* jerryx_handler_register_global */
+
+/**
+ * Set multiple properties on a target object.
+ *
+ * The functions are an array of (name, property value) pairs and
+ * this list must end with a (NULL, 0) entry.
+ *
+ * Notes:
+ *  - Each property value in the input array is released after a successful property registration.
+ *  - The method `jerryx_property_entry_cleanup` must be called if there is any failed registration
+ *    to release the values in the entries array.
+ *
+ * @return jerryx_register_result struct - if everything is ok with the (undefined, property entry count) values.
+ *         In case of error the (error object, registered property count) pair.
+ */
+jerryx_register_result
+jerryx_set_properties (const jerry_value_t target_object, /**< target object */
+                       const jerryx_property_entry entries[]) /**< array of method entries */
+{
+  uint32_t idx = 0;
```

Comment:
The `target_object` and each entry's value should be check whether they are error values or not.

Reply:
AFAIK, that would be too much, we'll rely on the `jerry_set_property` to generate an error on such cases and mainly on the user to not pass errors.